test/gleam_json_js_ffi_test.gleam

```diff
@@ -0,0 +1,72 @@
+import gleeunit
+import gleam/json.{DecodeError, UnexpectedByte, UnexpectedEndOfInput}
+import gleeunit/should
+
+if javascript {
+  pub fn main() {
+    gleeunit.main()
+  }
+}
```

Comment:
I think we want to remove this and instead put the tests in a conditional compilation block